lexicons/com/atproto/admin/moderationAction.json

```diff
@@ -32,6 +32,24 @@
         "did": {"type": "string"}
       }
     },
+    "subjectRecord": {
```

Comment:
does it make sense to unify these with the similar types in `com.atproto.repo.report`?

Reply:
Yeah, I think so, but there are a few things happening here that I wondered if we'd want to untangle at the same time:
 - The `{ did, collection, rkey, cid? }` shape is shared by `deleteRecord` and `putRecord` (though they don't carve it out into `subject`).
 - The `{ uri, cid }` pair is essentially a `com.atproto.repo.strongRef`, but there are still other methods that aren't using it such as the `*Record` outputs.  I almost used it, but it felt a bit funny to mix `com.atproto.repo.strongRef` in a union alongside something more single-use like `com.atproto.admin.moderationAction#subjectRepo`.
 - `{ did }` is arguably used by `batchWrite` and some other places to reference a repo.

I could see these as three shareable types: `recordRef`, `strongRef`, and `repoRef` respectively.  If we want to share some types across `com.atproto.*`, do you think we'd want to create a `com.atproto.system` namespace for them?  Or place them under `com.atproto.repo`?

Reply:
ah strongRef is already in `.repo` so lets just stick them there